example/06-volumesnapshotlocation.yaml

```diff
@@ -67,9 +67,23 @@ spec:
 #     bucket: openebs-velero-example
 #     prefix: cstor
 #     provider: aws
+#
+#     # The AWS region where the bucket is located.
```

Comment:
n_: specifying region of `minio` is confusing as it means that you are using minio product. Can this be changed to real region names used by AWS - like `us-east` or something like that?

Reply:
Can this example be split into two sections. This looks more like a code with if/else blocks. 

It will be clear for end users if we have this split as follows:

#When using AWS
# put the sample spec

#When using GCP
# put the sample spec

#When using Minio
# put the sample spec

Reply:
We are already having two different config for aws and minio. I will remove the mention of minio from minio section.